.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: Release
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  PANDOC_VERSION: 3.1.1.0
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - release
+
+jobs:
+  release-pr:
+    # guard against running on forks
+    if: github.repository == 'TTT-2/TTT2'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Check if PR was merged
+        uses: actions-ecosystem/action-release-label@v1
+        id: release-label
+        if: ${{ github.event.pull_request.merged == true }}
+
+      - name: Get latest tag if a release label exists on last merged PR
+        uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+        if: ${{ steps.release-label.outputs.level != null }}
```

Comment:
Is an action skipped or does the whole workflow fail on a failed if condition?
If it does fail, I wonder why we recheck it in other actions.

Reply:
only this specific step is skipped.

i'm not sure if i want all of these steps in a single job like it is now but it is the way i went with for now.
Additionally i need to double check all if-conditions anyway